erasure_code/erasure_code_update_test.c

```diff
@@ -59,6 +59,11 @@
 
 #define EFENCE_TEST_MAX_SIZE 0x100
 
+#if HAVE_RVV
+#define EC_ALIGNED_ADDR
+#define ALIGN_SIZE 32
```

Comment:
Can you move this ALIGN_SIZE to line 38? So:

`
#if HAVE_RVV
#define ALIGN_SIZE 32
#else
#define ALIGN_SIZE 16
#endif
`